cristin-person/src/test/java/no/unit/nva/cristin/person/handler/userAdministrator.feature

```diff
@@ -33,20 +33,18 @@ Feature: API tests for Cristin Person fetch
     Then status 200
     And response.NationalIdentificationNumber != '#present'
 
-  Scenario: Fetch returns status 403 Forbidden when not authorized
-    Given path '/person/identityNumber'
-    And header Authorization = 'Bearer ' + simpleUserToken
-    And request { type : NationalIdentificationNumber, value : '07117631634' }
-    When method POST
-    Then status 403
-    And match response.title == 'Forbidden'
-    And match response.status == 403
+  Scenario: Fetch returns unclassified person data when not authenticated nor authorized
+    Given path '/person/' + samplePersonIdentifier
+    When method GET
+    Then status 200
+    And response.NationalIdentificationNumber != '#present'
 
-  Scenario: Fetch returns 401 Unauthorized when not authenticated
+  Scenario: Fetch returns unclassified person data when not authenticated nor authorized with strange token
     Given path '/person/' + samplePersonIdentifier
-    And request
+    * header Authorization = 'Bearer and.just-a.silly-text-for-token'
```

Comment:
Should we return Forbidden or OK when user sends token? OK without token is OK but should we explicitly deny a user access when with an invalid token?